ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx

```diff
@@ -30,7 +30,7 @@ export class Dropdown {
   render() {
     return (
       <div class="ontotext-yasgui-dropdown">
-        <button class={`ontotext-yasgui-dropdown-button ${this.open ? 'icon-caret-up-after' : ' icon-caret-down-after'}`}
+        <button class={`ontotext-dropdown-button ${this.open ? 'icon-caret-up-after' : ' icon-caret-down-after'}`}
```

Comment:
Can you please remove the `yasgui` from the dropdown below as well

Reply:
done